src/components/reusables/UserInfoBar/UserInfoBar.tsx

```diff
@@ -0,0 +1,48 @@
+import { SvgIcon } from '@altinn/altinn-design-system';
+import * as React from 'react';
+import { Office1Filled } from '@navikt/ds-icons';
+import { useEffect } from 'react';
+
+import { ReactComponent as AltinnLogo } from '@/assets/AltinnTextLogo.svg';
+import { useAppDispatch, useAppSelector } from '@/rtk/app/hooks';
+import { fetchUserInfo, fetchReportee } from '@/rtk/features/userInfo/userInfoSlice';
+
+import classes from './UserInfoBar.module.css';
+
+export const UserInfoBar = () => {
+  const userInfoName = useAppSelector((state) => state.userInfo.name);
+  const reporteeName = useAppSelector((state) => state.userInfo.reporteeName);
+  const userLoading = useAppSelector((state) => state.userInfo.userLoading);
+  const reporteeLoading = useAppSelector((state) => state.userInfo.reporteeLoading);
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    if (userLoading || reporteeLoading) {
+      void dispatch(fetchUserInfo());
+      void dispatch(fetchReportee());
+    }
+  }, []);
+
+  return (
+    <div className={classes.userInfoBarMargin}>
+      <div className={classes.userInfoBar}>
+        <div className={classes.altinnLogo}>
+          <AltinnLogo />
+        </div>
+        <div className={classes.userInfoContent}>
+          <div>
+            {userInfoName && <h5 className={classes.userText}>{userInfoName}</h5>}
+            {reporteeName && <h5 className={classes.userText}>for {reporteeName}</h5>}
+          </div>
```

Comment:
Long company names pushes the users name far to the center of the screeen
![image](https://user-images.githubusercontent.com/22012997/221183150-97ce4d59-91ad-4aef-873b-236eb8bef9e7.png)

Perhaps the text should be aligned right as in the studio-apps?
![image](https://user-images.githubusercontent.com/22012997/221183337-7a41a53a-d52a-42ac-941e-03c7d417e310.png)